src/foundry/common/documents/user.d.mts

```diff
@@ -57,12 +63,11 @@ declare abstract class BaseUser extends Document<"User", BaseUser.Schema, any> {
   /**
    * Test whether the User is able to perform a certain permission action.
    * The provided permission string may pertain to an explicit permission setting or a named user role.
-   * Alternatively, Gamemaster users are assumed to be allowed to take all actions.
```

Comment:
I still see this line on v13.341. Was it... temporarily removed in v12 or something?